error: this statement may fall through

I was trying to install [polybar](https://github.com/jaagr/polybar) which uses jsoncpp as a dependency but ran into an error while compiling which seem to be coming from jsoncpp

```
[  2%] Building CXX object lib/i3ipcpp/libs/jsoncpp-1.7.7/src/lib_json/CMakeFiles/jsoncpp_lib_static.dir/json_reader.cpp.o
/tmp/polybar/lib/i3ipcpp/libs/jsoncpp-1.7.7/src/lib_json/json_reader.cpp: In member function ‘bool Json::OurReader::readToken(Json::OurReader::Token&)’:
/tmp/polybar/lib/i3ipcpp/libs/jsoncpp-1.7.7/src/lib_json/json_reader.cpp:1230:5: error: this statement may fall through [-Werror=implicit-fallthrough=]
     if (features_.allowSingleQuotes_) {
     ^~
/tmp/polybar/lib/i3ipcpp/libs/jsoncpp-1.7.7/src/lib_json/json_reader.cpp:1235:3: note: here
   case '/':
   ^~~~
cc1plus: all warnings being treated as errors
lib/i3ipcpp/libs/jsoncpp-1.7.7/src/lib_json/CMakeFiles/jsoncpp_lib_static.dir/build.make:62: recipe for target 'lib/i3ipcpp/libs/jsoncpp-1.7.7/src/lib_json/CMakeFiles/jsoncpp_lib_static.dir/json_reader.cpp.o' failed
make[2]: *** [lib/i3ipcpp/libs/jsoncpp-1.7.7/src/lib_json/CMakeFiles/jsoncpp_lib_static.dir/json_reader.cpp.o] Error 1
CMakeFiles/Makefile2:541: recipe for target 'lib/i3ipcpp/libs/jsoncpp-1.7.7/src/lib_json/CMakeFiles/jsoncpp_lib_static.dir/all' failed
make[1]: *** [lib/i3ipcpp/libs/jsoncpp-1.7.7/src/lib_json/CMakeFiles/jsoncpp_lib_static.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2

```

--- a/src/lib_json/json_reader.cpp
+++ b/src/lib_json/json_reader.cpp
@@ -1237,7 +1237,7 @@ bool OurReader::readToken(Token& token) {
     token.type_ = tokenString;
     ok = readStringSingleQuote();
     break;
-    } // else continue
+    } // else fall through
   case '/':
     token.type_ = tokenComment;
     ok = readComment();

